src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -335,8 +360,8 @@ def label
 
       def help
         _("<p><b>Enable Quota Support:</b>\n" \
-        "The file system is mounted with user quotas enabled.\n" \
-        "Default is false.</p>\n")
+          "The file system is mounted with user quotas enabled.\n" \
+          "Default is false.</p>\n")
```

Comment:
As HTML code is generated, I would say the last \n is leftover because `</p>` implies new line. An also, maybe we should use `<br />` instead of \n ?

Reply:
That "\n" is unnecessary, right. "<p>...</p>" takes care of that just fine.